homeassistant/components/gc100.py

```diff
@@ -0,0 +1,63 @@
+"""
+Support for controlling Global Cache gc100.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/gc100/
+"""
+
+import logging
+
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_STOP, CONF_HOST, CONF_PORT)
+
+REQUIREMENTS = ['python-gc100==1.0.1a']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'gc100'
+
+DATA_GC100 = 'gc100'
+
```

Comment:
https://home-assistant.io/developers/code_review_component/#configuration[Configuration validation](url) is missing.